Singularity/Singularity/Platforms/Road.cs

```diff
@@ -115,5 +114,12 @@ public float GetCost()
         {
             return Vector2.Distance(Source, Destination);
         }
+
+        public bool Die()
+        {
```

Comment:
What about units on the road? How do we detect them?

Reply:
good question tbh. Simple answer: No idea.
Long answer:
Since there's no reference to them from the Road, We can't do anything about that there. Since they're not on a platform or anything, we can't do anything there either. So it'd probably need to happen in the StructureMap somwhere, testing where units are headed towards. We do not need to delete them, but they then have tasks sending them there (or from), so they'd automatically get new one's. Maybe pathfinding fails then because the platform they're supposed to be at has been gone missing. But, They should not die probably, and this is not going to happen. Still, them being off the grid we should implement a 'speedup' for them or sth to get on it again (similiar to Achikaps)

Still, other Issue.